### PR TITLE
fixed broken makefile rule

### PR DESCRIPTION
**Pull request by rmodrak**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 227](http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/227).

File: src/tomography/postprocess_sensitivity_kernels/rules.mk [hunk 1]
```diff
@@ -41,8 +41,8 @@
 #========================================================================
 
 ## compilation directories
-S := ${S_TOP}/src/postprocess
-$(postprocess_OBJECTS): S := ${S_TOP}/src/postprocess
+S := ${S_TOP}/src/tomography/postprocess_sensitivity_kernels
+$(postprocess_OBJECTS): S := ${S_TOP}/src/postprocess_sensitivity_kernels
```
Comment (QuLogic) [inline src/tomography/postprocess_sensitivity_kernels/rules.mk]: You missed a `tomography` in this line, I think.

**Comment by komatits** [comment]
accept

On 05/14/2015 08:35 AM, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 227
> <http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/227>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/349#issuecomment-101936883>.Web
> Bug from
> https://github.com/notifications/beacon/AFjDKXwyMlE9OTB5CKxnlgjLPgNtoy3xks5oJDmYgaJpZM4EaGzz.gif
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

**Pull request merged**